python-sdk/src/astro/databases/base.py

```diff
@@ -305,50 +360,65 @@ def load_file_to_table(
         :param enable_native_fallback: Use enable_native_fallback=True to fall back to default transfer
         """
         normalize_config = normalize_config or {}
+
+        self.create_schema_table_if_needed(
+            file=input_file,
+            table=output_table,
+            columns_names_capitalization=columns_names_capitalization,
+            if_exists=if_exists,
+        )
+
+        if use_native_support and self.is_native_load_file_available(
+            source_file=input_file, target_table=output_table
+        ):
+
+            self.load_file_to_table_natively_with_fallback(
+                source_file=input_file,
+                target_table=output_table,
+                if_exists="append",
+                normalize_config=normalize_config,
+                native_support_kwargs=native_support_kwargs,
+                enable_native_fallback=enable_native_fallback,
+                chunk_size=chunk_size,
+            )
+        else:
+            self.load_file_to_table_using_pandas(
+                input_file=input_file,
```

Comment:
@sunank200 What will happen in pandas path? I'm not seeing the entire code here but all I see it the input_fiel getting passed to `load_file_to_table_using_pandas` will this work?

Reply:
never mind got the implementation and you are resolving the path there.